prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -50,13 +50,12 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: alpine:3.15.0
+      - image: bash:alpine3.14
         command:
-        - /bin/sh 
+        - bash
         args:
         - -c
-        - cd ./manifests/gcp_marketplace/test
-        - sh ./presubmit.sh
+        - bash ./manifests/gcp_marketplace/test/presubmit.sh
```

Comment:
Could this be the following instead if we stick with the same image?
```yaml
command:
- bash
args:
- ./manifests/gcp_marketplace/test/presubmit.sh
```

`bash -c bash ./script.sh` seems redundant.

Reply:
That is a good point, removing the `-c` for removing redundant argument.